app/components/gh-ed-preview.js

```diff
@@ -55,6 +55,15 @@ export default Component.extend({
         let template = document.createElement('template');
         template.innerHTML = html;
         let fragment = template.content;
+
+        if (!fragment) {
```

Comment:
So my original attempt at this used a polyfill, but because we manually create the `template` element using `document.createElement`, the only polyfill that would have worked is to override document.createElement entirely. This was a bit tricky, especially given that we set the `innerHTML` of the template before we actually get the content itself. Attempts at using the ES5 `get` method were unsuccessful. Also, determining where to load the polyfill in the stack of JS files was problematic. Ultimately this was easier, and avoided overriding such a critical method.